db/seeds.rb

```diff
@@ -12,3 +12,9 @@
   Doorkeeper::Application.create(name: "iOS client", redirect_uri: "", scopes: "")
   Doorkeeper::Application.create(name: "Android client", redirect_uri: "", scopes: "")
 end
+
+if Rails.env.development?
+  user = User.where(email: 'admin@nimblehq.co').first || Fabricate(:admin, email: 'admin@nimblehq.co')
```

Comment:
Using `exists?` would be more simple and performant 
```suggestion
  user = User.exists?(email: "admin@nimblehq.co") || Fabricate(:admin, email: 'admin@nimblehq.co')
```

Reply:
Good feedback as it reminded me about the `exist?` method :) :) 

In this scenario I expected to assign 100 keywords on the admin user, **and** create this user if not done yet.

If I use exist? it will return `true` and assign `true` to user, isn't it? 
What do you think @junan ? :)

Reply:
Ahh, ya. Its return `true` but you need to add a user.

Alternatively, you can use `first_or_create` method also like bellow like bellow. This is optional.
```ruby
 user = User.where(email: 'admin@nimblehq.co').first_or_create(Fabricate.attributes_for(:admin).except(:email))
 ```

Reply:
Interesting, it prevent to think about the logic behind `||` ;-) I'll take that then :)  
Done~